app/src/main/java/com/example/fuji/CreateBoardFragment.kt

```diff
@@ -1,18 +1,49 @@
 package com.example.fuji
 
 import android.os.Bundle
+import android.os.Handler
+import android.util.Log
 import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Button
+import android.widget.EditText
 import android.widget.ImageView
-import androidx.recyclerview.widget.LinearLayoutManager
-import androidx.recyclerview.widget.RecyclerView
-import com.example.fuji.models.Board
+import android.widget.Toast
+import com.google.firebase.firestore.ktx.firestore
+import com.google.firebase.ktx.Firebase
+
 
 
 class CreateBoardFragment : Fragment() {
 
+    ///////////////////////////////////////////////////////////////////////////////////////////////
+    ///////////////////////////////////START OF FIREBASE///////////////////////////////////////////
+
+    private val db = Firebase.firestore
+
+    private fun addBoard(boardName: String) {
+        val newBoard = hashMapOf(
+                "Title" to boardName,
```

Comment:
I think you can drop this. Our Board model only has `Name` on it

Reply:
The boards seem to be using title instead of name (just checked and only name leaves null for the board). Fixed to just Title

Reply:
good catch. I knew it was one or the other.